### ISSUE TITLE
Is not possible to install it on Ubuntu 20.04?

### ISSUE THREAD

**Issue by leillo1975** [kozec/sc-controller#553]
Please, add a repo for this LTS Ubuntu version. I try to install the snap, but don't work.

**Comment by FreeTheSmoke** [comment]
I've installed on Ubuntu 20.04
I think this program was my first time I tried build & install of software see instructions @
https://github.com/kozec/sc-controller/tree/v0.4.7#building-the-package-by-yourself

**Comment by Virusmater** [comment]
Some things will not work out of the box. e.g sc-controller is using python2, but evdev package for python2 isn't available by default in 20.04

**Comment by FreeTheSmoke** [comment]
oh, that's true, I had to dig around for some GTK lib's to make the keyboard keys show in the OSD.
  My main point to @leillo1975 was that it is possible to make this happen yourself, rather than waiting for others, and this is a pretty great package to have a try at compiling from if they haven't done it before. I'm happy to provide some guidance (although not a software pro by any means)

**Comment by leillo1975** [comment]
> Some things will not work out of the box. e.g sc-controller is using python2, but evdev package for python2 isn't available by default in 20.04

I have the same problem. I tried also the AppImage and Snap, but the first don't run and the second don't detect my Steam Controller. Is a pity that this awesome App don't have official support for a LTS version.... I can only hope and trust that the developer will take action

**Comment by solbero** [comment]
Until there is a compatible package for 20.04 I would like to compile sc-controller myself. I tried the snap and set the appropriate apparmour exceptions without success. Would you mind, @FreeTheSmoke , giving some pointers on how you compiled sc-controller on 20.04?

**Comment by LinuxJoeUser** [comment]
Re: Building the package by yourself, the Ubuntu package names are:

Dependencies
	*  _PyGObject_ is called `python-gobject-2` in Ubuntu
	*  _python-gi-cairo_ and _gir1.2-rsvg-2.0_ are called same in Ubuntu
	*  _setuptools_ is called `python-setuptools` in Ubuntu
	*  _python-pylibacl_ is called same in Ubuntu
	* _ python-evdev_ is called same in Ubuntu

Also, if you get an error about "headers" when you try to build/install add either
`python-dev` or `zlib1g-dev` --- I can't remember which --- but I add both.

**Comment by leillo1975** [comment]
"is not possible to find python-evdev"
"is not possible to find python-dev-is-python"

**Comment by solbero** [comment]
I had to download and install `pip2` manually from [https://pip.pypa.io/en/stable/installing/](https://pip.pypa.io/en/stable/installing/). Then I used `pip2` to build the missing Python 2 packages in Ubuntu 20.04.

**Comment by special-squad** [comment]
im having the same issue,wish he made another appimage for the newest version

**Comment by Ryochan7** [comment]
The following AppImage is confirmed to work on Ubuntu 20.04. It is based on the Python 3 fork.

SC_Controller-x86_64_20200707.AppImage:
https://drive.google.com/file/d/16QHJ4d5eaCYCYPdmFQ96DWJLh81L7gX8/view?usp=sharing

**Comment by leillo1975** [comment]
> The following AppImage is confirmed to work on Ubuntu 20.04. It is based on the Python 3 fork.
> 
> SC_Controller-x86_64_20200707.AppImage:
> https://drive.google.com/file/d/16QHJ4d5eaCYCYPdmFQ96DWJLh81L7gX8/view?usp=sharing

It Works! Great!. I think this must be uploaded here as official release

**Comment by leillo1975** [comment]
> The following AppImage is confirmed to work on Ubuntu 20.04. It is based on the Python 3 fork.
> 
> SC_Controller-x86_64_20200707.AppImage:
> https://drive.google.com/file/d/16QHJ4d5eaCYCYPdmFQ96DWJLh81L7gX8/view?usp=sharing

Where I can find the Python3 fork?

**Comment by mindbender444** [comment]
> > The following AppImage is confirmed to work on Ubuntu 20.04. It is based on the Python 3 fork.
> > SC_Controller-x86_64_20200707.AppImage:
> > https://drive.google.com/file/d/16QHJ4d5eaCYCYPdmFQ96DWJLh81L7gX8/view?usp=sharing
> 
> It Works! Great!. I think this must be uploaded here as official release

Yep, working well for me. Thanks for this. This needs to be made available on the front github page as an official release.

**Comment by krotax01** [comment]
> The following AppImage is confirmed to work on Ubuntu 20.04. It is based on the Python 3 fork.
> 
> SC_Controller-x86_64_20200707.AppImage:
> https://drive.google.com/file/d/16QHJ4d5eaCYCYPdmFQ96DWJLh81L7gX8/view?usp=sharing

With that AppImage I get only the tray icon, but no GUI, even lunching it with the 'gui' command. The tray icon provides only a button to enable/disable emulation, About and Quit options. Any help?

**Comment by mcphail** [comment]
I've been working on the snap version. It should now be working, to some extent, on 20.04 and 20.10. Certainly it "works for me" on all of my machines running 20.04 using several Steam Controllers and the controller dongle. I haven't implemented bluetooth support (yet) and haven't tried with any non-Steam controllers.

Give it a try and see if it works for you. Feedback is welcome on my sc-controller-snap repo.
